src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -1427,9 +1427,9 @@ void func_80832630(PlayState* play) {
     }
 }
 
-void func_8083264C(Player* this, s32 arg1, s32 arg2, s32 arg3, s32 arg4) {
+void Player_RumbleRequest(Player* this, s32 sourceStrength, s32 duration, s32 decreaseRate, s32 distSq) {
```

Comment:
It's pretty minor but I feel like `Player_RequestRumble` would be more natural

Reply:
fwiw I like `Player_RumbleRequest` as the pattern `Player_<system><thing>` Player's wrappers may follow
That said I agree `Player_RequestRumble` is more natural
so idk :shrug:

Reply:
In this case it's very much a single function by itself, so I don't think it matters too much.

Reply:
yea, will +1 RequestRumble personally